main/src/main/res/values/strings_not_translatable.xml

```diff
@@ -75,21 +75,18 @@
     <!-- recent contributors, meaning: last 12 months (app.) -->
     <!-- one line per contributor, format is: <name>|<link>|<contribution identifiers>| -->
     <!-- (contribution identifiers are: c=code, d=documentation, g=graphics, i=infrastructure & CI, p=project leader, s=support, t=testing) -->
-    <!-- currently listed: contributions 2023-01 to 2023-12 -->
+    <!-- currently listed: contributions 2024-07 to 2024-07 -->
```

Comment:
Really just from this month?

Reply:
oops, no - should be "last 12 months" (2023-07 to 2024-07) - good catch